src/test/kotlin/racingcar/domain/racing/CarsHistoryTest.kt

```diff
@@ -0,0 +1,45 @@
+package racingcar.domain.racing
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import racingcar.domain.car.Car
+import racingcar.domain.car.CarAction
+import racingcar.domain.car.Cars
+
+internal class CarsHistoryTest {
+
+    @Test
+    internal fun `자동차가 움직인 히스토리를 동적으로 추가할 수 있다`() {
+        val carsHistory = CarsHistory()
+        val cars = Cars(listOf())
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        val cars = Cars(emptyList())
```

Reply:
https://github.com/next-step/kotlin-racingcar/pull/238/commits/eab1e962691837ab74544d92ba105a3bcec3204f 사용하는 곳에서 전부 변경했습니다